config/plugins/interactive_environments/cellxgene/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:bionic
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+RUN apt-get update && \
```

Comment:
If you want to be based on ubuntu, then both RUN commands should be combined and unneeded packages should be removed after installation.

```
RUN apt-get update && apt-get install -y \
    build-essential \
    libxml2-dev \
    python3-dev \
    python3-pip \
    zlib1g-dev \
 && pip3 install cellxgene \
 && apt-get remove zlib1g-dev python3-dev ... \
 && rm -rf /var/lib/apt/lists/*
```

Reply:
Derived from python-slim now, I don't think it is getting much smaller...